linera-sdk/Cargo.toml

```diff
@@ -51,11 +58,10 @@ linera-chain = { workspace = true, features = ["metrics"] }
 linera-core = { workspace = true, features = ["test", "metrics", "wasmer"] }
 linera-execution = { workspace = true, features = ["fs", "metrics", "wasmer"] }
 linera-storage = { workspace = true, features = ["metrics", "wasmer"] }
-linera-witty = { workspace = true, features = ["test"] }
+linera-witty = { workspace = true, features = ["wasmer", "test"] }
```

Comment:
Isn't `wasmer` optional?

Reply:
Yes, I meant to use `wasmtime` here, which is needed by the `linera-wasm-test-runner`. Maybe I should just add it as a `required-features` for the binary instead? :thinking: